addons/sale_pdf_quote_builder/models/product_document.py

```diff
@@ -8,16 +8,21 @@ class ProductDocument(models.Model):
     _inherit = 'product.document'
 
     attached_on = fields.Selection(
-        selection_add=[('inside', "Inside quote")],
+        selection_add=[('inside', "Inside quote pdf")],
         help="Allows you to share the document with your customers within a sale.\n"
-             "Leave it empty if you don't want to share this document with sales customer.\n"
-             "Quotation: the document will be sent to and accessible by customers at any time.\n"
+             "On quote: the document will be sent to and accessible by customers at any time.\n"
              "e.g. this option can be useful to share Product description files.\n"
-             "Confirmed order: the document will be sent to and accessible by customers.\n"
+             "On confirm order: the document will be sent to and accessible by customers.\n"
              "e.g. this option can be useful to share User Manual or digital content bought"
              " on ecommerce. \n"
-             "Inside quote: The document will be included in the pdf of the quotation between the "
+             "Inside quote pdf: The document will be included in the pdf of the quotation between the "
              "header pages and the quote table. ",
+        ondelete={
+            'hidden': 'set default',
+            'quotation': 'set default',
+            'sale_order': 'set default',
+            'inside': 'set default',
```

Comment:
```suggestion
            'inside': 'set default',
```
No need to put again all the possibilities =] (wish it'd be the same for the help message... :cry: )